### PR TITLE
community updates for 2021-12-01

### PR DESCRIPTION
**Pull request by ericseppanen**
Consider this PR a work in progress until Tuesday midnight PDT; I'll keep adding anything the shows up during the rest of the day.

### CODE REVIEW

**Comment by ericseppanen** [comment]
I'm done trawling for new material, consider this PR complete.

File: draft/2021-12-01-this-week-in-rust.md [hunk 1]
```diff
@@ -16,22 +16,50 @@ If you find any errors in this week's issue, [please submit a PR](https://github
...
+* [A prioritised micro-batch scheduler in rust](https://njk.onl/blog/gaffer.html)
+* [Go footguns: Go Defer and Rust Drop](https://dev.to/mark_saward/go-footguns-go-defer-and-rust-drop-17af)
+* [LLVM internals, part 4: attributes and attribute groups](https://blog.yossarian.net/2021/11/29/LLVM-internals-part-4-attributes-and-attribute-groups)
+* [Implementing Raft for Browsers with Rust and WebRTC](https://eevans.co/blog/wraft/)
+* [Hubris and Humility](https://oxide.computer/blog/hubris-and-humility)
+* [audio] [Game Development with Herbert Wolverson](https://rustacean-station.org/episode/048-herbert-wolverson/)
+* [video] [Rust Linz, November 2021 - Advent of Code by Rainer Stropek with Fred Morcos](https://www.youtube.com/watch?v=bejThbzAYOA)
+* [video] [Rust Linz, November 2021 - Serde Shenanigans by Armin Ronacher](https://www.youtube.com/watch?v=UhZGYS13twc)
+* [video] [Rust Linz, November 2021 - WebAssembly beyond the browser by Thorsten Hans](https://www.youtube.com/watch?v=o6qe4lw9-8c)
+* [video] [Rust 2021 Edition Explained](https://www.youtube.com/watch?v=uIeIBlB0JFw)
```
Comment (extrawurst) [inline draft/2021-12-01-this-week-in-rust.md:40]: the YouTube links are not shortened

Comment (extrawurst) [reply draft/2021-12-01-this-week-in-rust.md:40]: hm I can't find it anymore where we discussed that but I thought we were supposed to use this `https://youtu.be/...` notation - but now I am not sure anymore

Comment (ericseppanen) [reply draft/2021-12-01-this-week-in-rust.md:40]: I thought we decided that we would replace things that redirect with whatever they redirect to. That's the way I wrote it in the README, anyway.

**Comment by luisvonmuller** [comment]
Damn, I was a little late. Can we add last week trasnlation to Brazilian 🇧🇷 <small>portuguese</small>?
https://github.com/luisvonmuller/Esta-Semana-Em-Rust/blob/main/%23418.md

File: draft/2021-12-01-this-week-in-rust.md (hunk 1 above)
Comment (extrawurst) [reply draft/2021-12-01-this-week-in-rust.md:40]: ok damn then I did it the wrong way around last week, nvm

**Comment by nellshamrell** [comment]
@luisvonmuller I'll get that in in a separate commit :)

**Comment by nellshamrell** [comment]
Also @luisvonmuller feel free to open a PR adding in the translation link in the future :)

**Comment by nellshamrell** [comment]
Thank you so much!

**Comment by luisvonmuller** [comment]
> Also @luisvonmuller feel free to open a PR adding in the translation link in the future :)



> Also @luisvonmuller feel free to open a PR adding in the translation link in the future :)

For sure, the next one I'll be here with a PR. (Today I was kinda busy, sorry 😮‍💨).

**Pull request merged**